src/test/ui/traits/wf-trait-object-maybe-bound.rs

```diff
@@ -0,0 +1,13 @@
+// The purpose of this test is to demonstrate that `?Sized` is allowed in trait objects
+// (thought it has no effect).
+
+type _0 = dyn ?Sized;
+//~^ ERROR at least one non-builtin trait is required for an object type [E0224]
```

Comment:
Speaking of this test... can you add one with just `let _: Box<dyn >` and make sure it fails?

Reply:
That's more to do with `dyn` syntax in general, so doesn't really belong in this test, I think.

Reply:
If you try `Box<dyn >` on nightly you will get the error message above tho. It is not rejected in the parser which we can verify with:
```rust
macro_rules! foo { ($x:ty) => {} }
foo!(Box<dyn >);

#[cfg(false)]
fn foo(_: Box<dyn>) {}
```